docs/install/upgrade-installation.md

```diff
@@ -49,16 +48,10 @@ kubectl get pods --namespace knative-serving
 kubectl get pods --namespace knative-eventing
 ```
 
-```bash
-kubectl get pods --namespace knative-monitoring
-```
-
 ### Upgrading plug-ins
 
 If you have a plug-in installed, make sure to upgrade it at the same time as
-you upgrade your Knative components. For example, if you have the
-monitoring plug-in installed, upgrade it alongside Knative Serving and Eventing.
-
+you upgrade your Knative components.
```

Comment:
Add an example of the network plugins (`net-contour` or `net-istio`)?

Reply:
Can you please open a new issue for this @evankanderson so we don't widen the scope of this PR too much? :slightly_smiling_face: 
Who can provide details for what to add here also?

Reply:
Sorry, we were removing the only example, I was suggesting a replacement example.

Reply:
This isn't blocking.